codestyle/README.md

```diff
@@ -0,0 +1,96 @@
+# EMF.cloud Codestyle
+
+Collection of common reusable resources for code formatting and code checks in EMF.cloud Java
+projects.
+
+## Code formatting
+
+The `eclipse` directory contains the EMF.cloud settings for formatting and clean-up of Java code as well as code templates. You can use these settings on a per-project basis by simply copying the `.settings` directory into your Eclipse project.
+
+Alternatively, these settings can also be configured globally. This can be done in the preferences ( Window -> Preferences -> Java -> Code Style) by creating a new Profile and import the settings from the corresponding .xml file of the `eclipse` directory.
```

Comment:
```suggestion
Alternatively, these settings can also be configured globally. This can be done in the preferences (Window > Preferences > Java > Code Style) by creating a new profile and import the settings from the corresponding .xml file of the `eclipse` directory.
```